NPMPack.ps1

```diff
@@ -0,0 +1,26 @@
+Switch ("$env:Build_SourceBranchName")
+{
+    "master" { $env:tag = "master"  }
+    "develop" { $env:tag = "develop" }
+    default { exit }
+}
+
+cd *.Typescript/package
+Set-Content -Value "//www.myget.org/F/hipapp/npm/:_authToken=$env:MyGetKey" -Path ./.npmrc
```

Comment:
Shouldn't this also use `$env:MyGetFeed`?
Also the value for "MyGetFeed" is different for NuGet and NPM packages, so I suggest to use two variables: "NuGetFeed" and "NPMFeed"